### ISSUE TITLE
package_heltec_esp8266_index.json issue

### ISSUE THREAD

**Issue by abubakas** [HelTecAutomation/Heltec_ESP8266#3]
Skipping contributed index file C:\Users\abubakas\AppData\Local\Arduino15\package_heltec_esp8266_index.json, parsing error occured:
com.fasterxml.jackson.databind.JsonMappingException: Unexpected character ('£' (code 163)): was expecting comma to separate ARRAY entries
 at [Source: java.io.FileInputStream@202df7; line: 44, column: 11] (through reference chain: ContributionsIndex["packages"]->java.util.ArrayList[0]->ContributedPackage["platforms"])

**Comment by brobergp** [comment]
Yeah, get the same complains when trying to install the board library. 
The Json file is messed up.

**Comment by endeebee** [comment]
Confirmed - discovered the same. Arduino 1.8.10 on MacOS

**Comment by piedade2007** [comment]
Same problem.
 
Where I got so far... 
copied the Json file to my PC and edit it to replace the reported line 44 wrong char for a comma. Then found there are also things missing, a  "]" and "}" at the end of the file.
Then setup a local web-server (eg. fenix-windows2.0 or Zwamp) and placed the new json with the corrections on the local web-server ( in my case the new link to feed on arduinoIDE configuration is as follows: http://localhost/package_heltec_esp8266_index.json ) the problem is although the json file now seems to be ok there's a lot of java errors being reported on the IDE console... If someone can push this further please share what you find. Thanks!
